### ISSUE TITLE
[Question] Change language

### ISSUE THREAD

**Issue by ehhc** [atom/node-spellchecker#100]
Hi there,
i have a question concerning your spell checker. 
Is there a (easy) option to change the language of the spellcheck without changing the language of the whole OS?
E.g. My (windows) computer is using German language. Nevertheless i would like to spellcheck notes i took using english (not always but sometimes).
Since i'm using a windows machine, starting the process with `SET LANG=EN` or something similar is unfortunately not an option.. Is there anything i can do?

Thanks in advance

**Comment by dmoonfire** [comment]
You have a couple options:

1. Add English as your second language in Windows (I'm assuming Windows 10).
2. Use `SET SPELLCHECKER_PREFER_HUNSPELL=1`, download an English dictionary, and point your locale path to it. This approach ignores all of the system dictionaries and only uses the ones it finds on disk.

**Comment by ehhc** [comment]
Hi,
assuming i use option 1.: How can i switch between the languages without restarting the app or changing the language of my OS?

2. and 3.) If i do so, i don't have the benefits coming from using the native spell check anymore, do i?

Thank you :)

**Comment by AlexandreKilian** [comment]
@ehhc did you figure it out? Even with german installed as a secondary language, it's just working for german…
as for option 2 or 3,  @dmoonfire how do I even switch to hunspell and set languages/install dictionaries? nothing about that in the docs, I browsed the code too, nothing there either

**Comment by ehhc** [comment]
@AlexandreKilian no, unfortunately i didn't figured it out. i used another project in the end.. :(

**Comment by AlexandreKilian** [comment]
@ehhc I figured it out after checking the code: `Spellchecker.setDictionary('de');`…

**Comment by ehhc** [comment]
okay.. thanks for the information :)
